MarkBind/markbind
# Use only forward slashes for paths during build

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] Enhancement to an existing feature


Resolves #628.



**What is the rationale for this request?**
When MarkBind generates paths during site build, it uses the default path separator of the OS it is running on (`\` on Windows, `/` on Mac/Linux). For consistency, MarkBind should use the same path seperator regardless of OS.

**What changes did you make? (Give an overview)**
I created a temporary `assets` object which has all the paths run through `ensurePosix` to build the template with all forward slashes its paths. I also modified the parser to generate paths with forward slashes for dynamic `panel` includes.

**Testing instructions:**
1. `markbind build` a site. The generated HTML should use forward slashes regardless of OS.
